### PR TITLE
add cicd concepts to class05

### PR DESCRIPTION
**Pull request by caiocezart**

### CODE REVIEW

File: classes/05class/cicd/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Continuous Integration / Continuous Deployment
+
+I need to add/delete changes to my running application as automated as possible.
+ 
+***Contents***
+- [Pipeline]()
+- [Continuous Integration]()
```
Comment (denstorti) [inline classes/05class/cicd/README.md]: Add Continuous Delivery?

**Comment by denstorti** [comment]
It looks good 👍

File: classes/05class/cicd/README.md (hunk 1 above)
Comment (caiocezart) [reply classes/05class/cicd/README.md]: tbh as our content is entry-level, I decided to abstract delivery into deployment as they overlap quite a bit.

Comment (caiocezart) [reply classes/05class/cicd/README.md]: and I think you review an old version that the summary was incomplete haha

**Pull request merged**